.circleci/config.yml

```
@@ -0,0 +1,16 @@
+version: 2.1
+
+orbs:
+  architect: giantswarm/architect@1.0.0
```

Please use this newer orb version, it contains an important fix regarding chart index urls

```suggestion
  architect: giantswarm/architect@2.2.0
```